apps/i18n/common/en_us.json

```diff
@@ -98,6 +98,7 @@
   "authorizeGoogleClassroomsText": "In order to sync with Google Classroom, Code.org must have up-to-date authorization to access your Google Classroom account.",
   "autoGenerated": "Auto-generated",
   "autolock": "Note: Stage auto-locks after 24 hours.",
+  "availableLessons": " Available in {numLessons} Lesson(s):",
```

Comment:
There's a little pluralization trick you can use here. For example,  "exactNumberOfBlocks": "Only {numBlocks, plural, one {1 block} other {# blocks}} used!",

Reply:
cool done!